keypom/keypom-airfoil
# Fydp min integrations

# Description
Integration for purchasing flow, working with the stripe and email workers, sending proper emails. At the moment, seller does not get notified when their ticket is sold. 


# How to test
Test out different buying and selling flows. 
1) buying w near, then reselling
2) buying with stripe then reselling.

seller notification emails can be turned on my uncommenting Events.tsx lines 740-771 and set the email to your own. Expected behaviour is purchasing when logged in should result in resells being deposited in your wallet while tickets bought when not logged in and resold will send you an email with a linkdrop for sale amount.
